bundle-core/src/main/java/net/discdd/utils/FileUtils.java

```diff
@@ -17,4 +31,43 @@ public static void createFileWithDefaultIfNeeded(Path path, byte[] bytes) throws
     public static void createEmptyFileIfNeeded(Path path) throws IOException {
         createFileWithDefaultIfNeeded(path, new byte[0]);
     }
+
+    public static OutputStream getFilePath(BundleDownloadResponse response, String receive_Directory) throws IOException {
+        String fileName = response.getMetadata().getBid();
+        File directoryReceive = new File(receive_Directory + File.separator + response.getMetadata().getSenderId());
```

Comment:
we try to avoid + with filenames. pass receiveDirectory as a Path and do receiveDirectory.resolve(response.getMetadata().getSenderId())